lib/src/features/my_profile/presentation/my_profile_page.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class MyProfilePage extends HookConsumerWidget {
```

Comment:
When you don't use hook or riverpod, please use Stateless Widget. Please check the other code as well.

Reply:
@mafreud fixed, thank you